cost/kubecost/cluster/kubecost_cluster_rightsizing_recommendations.pt

```diff
@@ -248,40 +283,43 @@ script "js_recommendations", type: "javascript" do
     "- Target Utilization: ", param_target_util, "%\n\n",
     "The above settings can be modified by editing the applied policy and changing the appropriate parameters."
   ].join('')
+  savings = Math.round(recommendation['monthlySavings']).toFixed(2)
 
   recommendationDetails = [
     "Modify settings for cluster ", accountID,
     " so that node count is set to ", recommendation['nodeCount'],
     " and pools are configured to match the Recommended Pools field."
   ].join('')
 
-  result = [{
+  result.push({
     accountID: accountID,
     accountName: accountName,
     strategy: selected_strategy,
-    totalNodeCount: ds_cluster_sizing['totalNodeCount'],
-    totalRAMGB: ds_cluster_sizing['totalRAMGB'],
-    totalVCPUs: ds_cluster_sizing['totalVCPUs'],
-    monthlyRate: Math.round(ds_cluster_sizing['monthlyRate'] * 1000) / 1000,
-    totalMonthlyCost: Math.round(recommendation['totalMonthlyCost'] * 1000) / 1000,
+    totalNodeCount: crs['totalNodeCount'],
+    totalRAMGB: crs['totalRAMGB'],
+    totalVCPUs: crs['totalVCPUs'],
+    monthlyRate: Math.round(crs['monthlyRate']).toFixed(2),
+    totalMonthlyCost: Math.round(recommendation['totalMonthlyCost']).toFixed(2),
     // Note: Savings is presented as a negative value. Hence why we multiply by a negative number to invert it
-    savings: Math.round(recommendation['monthlySavings'] * -1000) / 1000,
+    savings: Math.round(recommendation['monthlySavings']).toFixed(2),
     savingsCurrency: ds_currency['symbol'],
     nodeCount: recommendation['nodeCount'],
     pools: JSON.stringify(recommendation['pools']),
     recommendationDetails: recommendationDetails,
     region: region,
     service: "Kubernetes",
-    policy_name: ds_applied_policy['name'],
     min_nodes: param_min_nodes,
     lookback: param_lookback,
     target_util: param_target_util,
     message: message,
+    policy_name: ds_applied_policy['name'],
     message_savings: [
       ds_currency['symbol'], ' ',
-      formatNumber(Math.round(recommendation['monthlySavings'] * -1000) / 1000, ds_currency['separator'])
+      formatNumber(Math.round(recommendation['monthlySavings']).toFixed(2), ds_currency['separator'])
```

Comment:
Why is this change being made? Removing a multiplication against a negative number is going to change the result. Is this intended?

Reply:
this was test code. I reverted this one  now the behaviur should be same